package.json

```diff
@@ -181,6 +181,7 @@
     "@deephaven/storage": "file:packages/storage",
     "@deephaven/stylelint-config": "file:packages/stylelint-config",
     "@deephaven/tsconfig": "file:packages/tsconfig",
-    "@deephaven/utils": "file:packages/utils"
+    "@deephaven/utils": "file:packages/utils",
+    "browserslist": "^4.23.2"
```

Comment:
Do we need this package? How is it being used?

Reply:
We do not need this package necessarily , but I thought it could be good for two reasons:
a. When the browser is not compatible, it shows a banner https://github.com/deephaven/deephaven-plugins/issues/510 (3rd tool here)
b. It could give us statistics about browser usage, can be leveraged in the future https://github.com/deephaven/deephaven-plugins/issues/510 (4th tool here)

However, if these benefits are not as valuable, could also be removed, no issues on my end.